### DISCUSSION TITLE
Installed nightly build but app no longer says APatch Next it just says APatch

### DISCUSSION THREAD

**Discussion by molitar** [bmax121/APatch#795] [Q&A]
I had to do a factory wipe so when I reinstall APatch I downloaded the nightly build previously it said APatch Next and I was able to load the module Cherish Peekaboo 1.5.3.

1. Downloaded Nightly Build https://nightly.link/bmax121/APatch/workflows/build/main/APatch.zip
2. Installed it only shows as APatch and not APatch Next
3. Patched Android 14 Factory Image boot.img
4. Fastboot flashed the new patched img titled apatch_patched_11003_0.11.1-dev_zvij.img
5. Reboot and entered in my SuperKey shows installed as 0.11.1 (11003) - Full
6. Verified Root is working but unable to install Cherish Peekaboo 1.5.0 or 1.5.3 still can only load 1.4.2

Why is it what worked before and I was able to install the newer Cherish is not working anymore? Also why is the app no longer showing as APatch Next so I know that I do have the correct nightly build downloaded?

**Comment by VisionR1** [comment]
The APatch Next is belong to the main, but still not merge it.
I don't know from where download it before, but not from here, maybe TG group.
![IMG_20241124_110037](https://github.com/user-attachments/assets/bd47c868-e3df-42e4-8099-5a67d5274d75)
